index.d.ts

```diff
@@ -44,7 +52,7 @@ declare namespace passport_openidconnect {
         userInfoURL: string,
         clientID: string,
         clientSecret: string,
-        callbackURL: string,
+        callbackURL?: string,
```

Comment:
Why does callbackURL need to be optional? I think it's a requirement of the Oauth2 spec?

Reply:
goooooood question. we're actually defaulting this to `/login` in atlas-runtime somewhere else, so I didn't see it was a needed param when I copied over from there. great catch. I updated that in scout (https://bitbucket.org/mdinsider/scout/pull-requests/137/callbackurl-is-a-required-param-in) and in this PR.